app/app.model.js

```diff
@@ -42,4 +42,23 @@ const selectArticleByID = (article_id) => {
     })
 }
 
-module.exports = { selectTopics, selectArticles, selectArticleByID }
\ No newline at end of file
+const selectCommentsByArticleId = (article_id) => {
+    if (/[^\d]/g.test(article_id)) {
```

Comment:
I like your thinking here, but it's not necessary. If the article id is not a number, it will automatically be rejected when it hits out sql statement. Please consult the error-handling notes for more info on this.